lib/msf/core/db_manager/event.rb

```diff
@@ -37,6 +37,7 @@ def events(opts)
     offset = opts.delete(:offset) || DEFAULT_OFFSET
 
     search_term = opts.delete(:search_term)
+    pagination_total = wspace.events.where(opts).length
```

Comment:
This should use `count` to avoid loading all the entries of the `events` table on every request.
```suggestion
    pagination_total = wspace.events.where(opts).count
```